### PR TITLE
feat: enhanced demo issuer UI

### PR DESCRIPTION
**Pull request by birtony**
image

image

image

image

Signed-off-by: Anton Biriukov <anton.biriukov@gendigital.com>

### CODE REVIEW

File: cmd/issuer-rest/static/preAuthorize.html [hunk 1]
```diff
@@ -8,22 +9,52 @@
...
     <script src="https://unpkg.com/web-credential-handler@1.0.1/dist/web-credential-handler.min.js"></script>
 
     <script src="https://cdnjs.cloudflare.com/ajax/libs/qrcodejs/1.0.0/qrcode.min.js"
             integrity="sha512-CNgIRecGo7nphbeZ04Sc13ka07paqdeTu0WR1IM4kNcpmBAUSHSQX0FslNhTDadL4O5SAGapGt4FodqL8My0mA=="
             crossorigin="anonymous" referrerpolicy="no-referrer"></script>
     <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" rel="stylesheet">
     <meta charset="UTF-8">
-    <title>VerifiedEmployee [Pre Auth]</title>
+    <title>TrustBloc Demo Issuer (Pre-Auth)</title>
```
Comment (rolsonquadras) [inline cmd/issuer-rest/static/preAuthorize.html:22]: Can we updated this to TrustBloc Demo Issuer (OpenID4CI Pre-Auth Code Flow) ? If it's long, we can add it in next line.

File: cmd/issuer-rest/static/preAuthorize.html [hunk 2]
```diff
@@ -33,43 +64,48 @@
...
-                       class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                       placeholder="{{.Pin}}" readonly>
+        <div class="w-full p-4" id="pinBox">
+            <div class="grid place-items-center">
+                <div>
+                    <label for="pin2" class="block mb-2 text-lg font-semibold text-gray-900">PIN</label>
+                    <button class="flex flex-row items-center" onclick="copyToClipboard('pin2')">
+                        <input type="text" id="pin2"
+                               class="cursor-pointer bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg text-center w-full p-2.5"
+                               value="{{.Pin}}" readonly disabled>
```
Comment (rolsonquadras) [inline cmd/issuer-rest/static/preAuthorize.html:80]: Can we add PIN text and PIN field value in same line ? Currently, it looks bit odd with them being in different lines.

File: cmd/issuer-rest/static/preAuthorize.html (hunk 1 above)
Comment (rolsonquadras) [reply cmd/issuer-rest/static/preAuthorize.html:22]: Ignore this. My point was on body content.

File: cmd/issuer-rest/static/preAuthorize.html [hunk 3]
```diff
@@ -33,43 +64,48 @@
...
         </div>
     </div>
+    <!-- Footer -->
+    <div id="demo-issuer-footer" class="mx-6 w-full py-6 px-15 text-center sticky bottom-0 z-20 border-t border-grey"></div>
 </div>
 </body>
 <script>
     $(".debug").hide()
 
+    // Setting footer value
```
Comment (DRK3) [inline cmd/issuer-rest/static/preAuthorize.html:106]: I think this comment `// Setting footer value` is redundant (I can see from the `#demo-issuer-footer` part of the line below that you're setting the footer)

File: cmd/issuer-rest/static/preAuthorize.html [hunk 4]
```diff
@@ -33,43 +64,48 @@
...
+        <div class="w-full p-4 debug">
+            <label>{{.URL}}</label>
+        </div>
+        <div class="w-full p-4">
+            <div class="grid place-items-center" id="successBlock" style="display: none">
+                <label>{{.SuccessText}}</label>
+            </div>
         </div>
     </div>
+    <!-- Footer -->
```
Comment (DRK3) [inline cmd/issuer-rest/static/preAuthorize.html:99]: Redundant comment (similar reasoning as my other PR comment)

**Review by DRK3 — APPROVED** [review]
Minor comments

File: cmd/issuer-rest/static/preAuthorize.html (hunk 2 above)
Comment (DRK3) [reply cmd/issuer-rest/static/preAuthorize.html:80]: Or maybe centered above the field?

**Review by talwinder50 — APPROVED** [review]

**Pull request closed without merging**